doc/configuration/configuration.rst

```diff
@@ -0,0 +1,10 @@
+.. _Configuration_details:
+
+##########################
+Electrometer Configuration
+##########################
+
+The Electrometer is configurable by changing filter states and types.
+The configuration files are located in the `ts_config_ocs repo <https://github.com/lsst-ts/ts_config_ocs>`_.
+
```

Comment:
The schema for the configuration file is located with the ts_electrometer repo.
(but make Schema the link)